src/irmin-pack/unix/snapshot.ml

```diff
@@ -49,7 +48,8 @@ module Make (Args : Args) = struct
       Index_unix.Make (Pack_index.Key) (Value_unit) (Index.Cache.Unbounded)
 
     type t = {
-      fm : File_manager.t;
+      fm : Fm.t;
```

Comment:
I suggest we don't use module aliases here (and keep `File_manager` instead of `Fm` - this helps to write shorter programs but doesn't help to read them later on.

Reply:
Not super important - so no need to have a tracking issue :-)